backend/db/schema.py

```diff
@@ -42,11 +45,43 @@ class SchoolDistrict(Base):
 
 
 class MoodleSetting(Base):
-    __tablename__ = 'moodle_setting'
+    __tablename__ = "moodle_setting"
     __table_args__ = (
-        UniqueConstraint('name'),
+        UniqueConstraint("name"),
     )
 
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     name: Mapped[str]
     value: Mapped[str]
+
+
+class Status(enum.Enum):
```

Comment:
Minor: I don't think this will change the migration, but it might be nice to call this `CourseStatus` so when we import into other files later it will be clear:

```python
from rope.db.schema import CourseStatus
```

Reply:
I have not upgraded my head yet. Should I go ahead and delete the migration I pushed above and generate a new one just to be safe?

Reply:
If you want, you can make the change and generate a new migration just to see if it detects any net difference (e.g. that `upgrade()` and `downgrade()` aren't just empty with `pass`). If it doesn't, you can just delete the new migration. If I was wrong and it does detect something difference, you can delete your existing migration and generate a new one.

Reply:
Now that I read :point_up: it actually sounds complicated. Maybe it's simpler to just use the process @P-Gill97 documented in the `README.md` :joy:

Reply:
That's what I normally do 😆 By changing the name of the class from `Status` to `CourseStatus` there was a name adjustment in the migration. I've provided an updated migration file in the latest commit.

Reply:
Updated in latest commit.